oxidation/libparsec/crates/types/tests/test_manifest.rs

```diff
@@ -513,3 +513,51 @@ fn test_file_manifest_verify(
         expected_result
     );
 }
+
+#[rstest]
+fn file_manifest_unverified_load() {
+    // Test the unverified_load() function on a file manifest.
+    // File manifest has been generated with Parsec 1.12, it is provided raw as it is a bit hard
+    // to generate (would need to create keys and all that). We're only interested in the
+    // deserialization of raw unciphered data to a FileManifest.
```

Comment:
```suggestion
/// Test the unverified_load() function on a file manifest.
/// File manifest has been generated with Parsec 1.12, it is provided raw as it is a bit hard
/// to generate (would need to create keys and all that). We're only interested in the
/// deserialization of raw unciphered data to a FileManifest.
fn file_manifest_unverified_load() {
```

Use doc-comment

Reply:
I just wanted to describe what the test does, not write documentation. It would make no sense to have this comment be exported in a doc.